src/cdk-experimental/scrolling/virtual-for-of.ts

```diff
@@ -35,7 +35,7 @@ import {CdkVirtualScrollViewport} from './virtual-scroll-viewport';
 /** The context for an item rendered by `CdkVirtualForOf` */
 export type CdkVirtualForOfContext<T> = {
   $implicit: T;
-  cdkVirtualForOf: NgIterable<T> | DataSource<T>;
+  cdkVirtualForOf: DataSource<T> | Observable<T[]> | NgIterable<T>;
```

Comment:
I'm confused about this WRT your comment about needing the length (the `NgIterable` here)

Reply:
I was originally going to support `Observable<NgIterable<T>>`, but decided there wasn't really much point. Still want to support plain `NgIterable<T>` so that you can pass all of the same stuff to `*cdkVirtualFor` that you would to `*ngFor`